### DISCUSSION TITLE
Support multiple variables in single interpolated string

### DISCUSSION THREAD

**Discussion by MellKam** [Alexandre-Fernandez/astro-i18n#67] [Ideas]
The current behaviour for multiple variables:

```json
{
	"hello": "a: {# a #}, b: {# b #}, c: {# c #}"
}
```

Generated type:

```ts
hello: { a?: unknown } | undefined;
```

Result:

```astro
{t("hello", { a: "A", b: "B", c: "C" })} 
// output: "a: A"
```

**Comment by Alexandre-Fernandez** [comment]
Indeed this should be added. I thought I already coded that, I'll try adding that asap.

**Comment by Alexandre-Fernandez** [comment]
https://github.com/Alexandre-Fernandez/astro-i18n/discussions/63

**Comment by Alexandre-Fernandez** [comment]
Btw I'll add this after I finish my current personal project, until then I'll only patch bugs.
